src/testhost.x86/DefaultEngineInvoker.cs

```diff
@@ -49,12 +49,14 @@ internal class DefaultEngineInvoker :
         public void Invoke(IDictionary<string, string> argsDictionary)
         {
             // Setup logging if enabled
-            string logFile;
-            if (argsDictionary.TryGetValue(LogFileArgument, out logFile))
+            if (argsDictionary.TryGetValue(LogFileArgument, out string logFile))
             {
                 EqtTrace.InitializeVerboseTrace(logFile);
             }
-
+            else
+            {
+                EqtTrace.DoNotInitailize = true;
```

Comment:
Unit tests for the case when DoNotInitialize is true to ensure trace is not initialzied?